www/js/control/AppStatusModal.tsx

```diff
@@ -0,0 +1,458 @@
+//component to view and manage permission settings
+import React, { useState, useEffect, useMemo } from "react";
+import { Modal,  useWindowDimensions, ScrollView } from "react-native";
+import { Dialog, Button, Text, useTheme } from 'react-native-paper';
+import { useTranslation } from "react-i18next";
+import PermissionItem from "../appstatus/PermissionItem";
+import useAppConfig from "../useAppConfig";
+import useAppStateChange from "../useAppStateChange";
+import ExplainPermissions from "../appstatus/ExplainPermissions";
+import AlertBar from "./AlertBar";
+
+const AppStatusModal = ({permitVis, setPermitVis, dialogStyle, settingsScope}) => {
+    const { t } = useTranslation();
+    const { colors } = useTheme();
+    const { appConfig, loading } = useAppConfig();
+
+    console.log("settings scope in app status modal", settingsScope);
+
+    const { height: windowHeight } = useWindowDimensions();
+    const [osver, setOsver] = useState(0);
+    const [platform, setPlatform] = useState<string>("");
+
+    const [error, setError] = useState<string>("");
+    const [errorVis, setErrorVis] = useState<boolean>(false);
+
+    const [explainVis, setExplainVis] = useState<boolean>(false);
+
+    const [backgroundRestricted, setBackgroundRestricted] = useState<boolean>(false);
+    const [allowBackgroundInstructions, setAllowBackgroundInstructions] = useState<Array<any>>([]);
```

Comment:
why are these two called out specially instead of being included with the other errors?
I don't see specific errors for location permissions, for example.

Reply:
Ah I see that these are for handling the samsung issue, but are currently unused
https://github.com/e-mission/e-mission-phone/pull/1009/files#diff-2c9b8aaf6321e412caf37d08c26037408558d62ed592f16fb0847b6cccd8639fR317

Reply:
done in [426c440](https://github.com/e-mission/e-mission-phone/pull/1028/commits/426c44069ef3446ba8ec33bad3dbdf5fec335adf)